documentation/metadata-editing.md

```diff
@@ -0,0 +1,87 @@
+# Editing metadata of resources
```

Comment:
General comments about the document:

* Talk about description resources instead of metadata resources, as the first is the term that is used in the spec. Specifically, we use the description resource to represent all the metadata we have on a resource.
* There should be a part somewhere explaining that we ignore container bodies when doing a PUT as that is something people might expect. And that we don't allow a PUT on an existing container, with some explanation why.
* It might be interesting to show the example output when doing a GET on the description resource of a standard description resource, and then perhaps a small N3 Patch (not SPARQL update) example that specifically adds an inbox to a resource, instead of the generic PATCH example that there is now, as that is probably something that would be useful for some people.
* Although I do say that we shouldn't list everything that isn't allowed, we should probably mention that DELETE does not work on metadata. Should in fact see this documentation partially as a "spec" document describing how description resources work for CSS, similar to how the protocol spec defines how the interactions with standard resources work.
* Any other exception mentioned in https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1027, but unless I missed one, I think this covers everything.